components/StatusBar.js

```diff
@@ -33,10 +31,11 @@ const useStyles = makeStyles(() => ({
   }
 }));
 
-const StatusBar = () => {
-  const { config: { build: { name, buildDate, version } } } = useContext(AppContext);
+const StatusBar = ({ config }) => {
```

Comment:
Why is this not read from context?

Reply:
It needs to be dynamic. I.e., read from ~/.well-known
That was the big research from yesterday. If I use a hook then the page flickers because first it returns a null config.
So this was the only way I could get async info prior to rendering:
```
Page.getInitialProps = async () => ({ config: await getDyanmicConfig() });
```